frameworks/helloworld/tests/test_folders.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import shakedown
+
+import sdk_install as install
+from tests.config import (
+    PACKAGE_NAME,
+    DEFAULT_TASK_COUNT,
+    check_running
+)
+
+FOLDERED_SERVICE_NAME = "/path/to/" + PACKAGE_NAME
+
+
+def uninstall_foldered():
+    install.uninstall(
+        FOLDERED_SERVICE_NAME,
+        package_name=PACKAGE_NAME,
+        role=FOLDERED_SERVICE_NAME.lstrip('/') + '-role',
+        principal=FOLDERED_SERVICE_NAME + '-principal',
+        zk='dcos-service-' + FOLDERED_SERVICE_NAME.lstrip('/').replace('/', '__'))
+
+
+def setup_module(module):
+    install.uninstall(PACKAGE_NAME)
+    uninstall_foldered()
+
+
+def teardown_module(module):
+    uninstall_foldered()
+
+
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_install_foldered():
```

Comment:
Done, added a CLI command (which effectively hits adminrouter) for both helloworld/kafka (the two that support it at the moment)